data/tlg0007/tlg082b/tlg0007.tlg082b.perseus-grc3.xml

```diff
@@ -184,190 +196,191 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
 </p>
 </div>
-<div n="8" subtype="section" type="textpart">
-<p rend="indent">Θάπτουσά τις τὸν υἱόν, ὡς γραΐδιον εὐτελὲς προσελθὸν αὐτῇ, <q type="unspecified">ὦ γύναι, τᾶς τύχας,</q> εἶπε, <q type="unspecified">νὴ.
+<div type="textpart" subtype="section" n="8">
+<p>Θάπτουσά τις τὸν υἱόν, ὡς γραΐδιον εὐτελὲς προσελθὸν αὐτῇ, <q type="unspecified">ὦ γύναι, τᾶς τύχας,</q> εἶπε, <q type="unspecified">νὴ.
 τὼ σιώ ἀλλὰ τᾶς καλᾶς γʼ,</q> ἔφη· <q type="unspecified">καὶ γὰρ αὐτὸν
 οὗ ἕνεκεν ἔτεκον, ἵνʼ ὑπὲρ τᾶς Σπάρτας ἀποθάνῃ,
-<milestone n="241d" unit="stephpage" xml:id="stephpage-241d" /> τοῦτό μοι συνέβη.</q>
+<milestone unit="stephpage" xml:id="stephpage-241d" n="241d"/> τοῦτό μοι συνέβη.</q>
 </p>
 </div>
-<div n="9" subtype="section" type="textpart">
-<p rend="indent">Σεμνυνομένης γυναικός τινος Ἰωνικῆς ἐπί τινι τῶν ἑαυτῆς ὑφασμάτων ὄντι πολυτελεῖ, Λάκαινα ἐπιδείξασα τοὺς τέτταρας υἱοὺς ὄντας κοσμιωτάτους, τοιαῦτα, ἔφη, δεῖ εἶναι τὰ τῆς καλῆς καὶ ἀγαθῆς γυναικὸς ἔργα καὶ ἐπὶ τούτοις ἐπαίρεσθαι καὶ μεγαλαυχεῖν. </p>
+<div type="textpart" subtype="section" n="9">
+<p>Σεμνυνομένης γυναικός τινος Ἰωνικῆς ἐπί τινι τῶν ἑαυτῆς ὑφασμάτων ὄντι πολυτελεῖ, Λάκαινα ἐπιδείξασα τοὺς τέτταρας υἱοὺς ὄντας κοσμιωτάτους, τοιαῦτα, ἔφη, δεῖ εἶναι τὰ τῆς καλῆς καὶ ἀγαθῆς γυναικὸς ἔργα καὶ ἐπὶ τούτοις ἐπαίρεσθαι καὶ μεγαλαυχεῖν. </p>
 </div>
-<div n="10" subtype="section" type="textpart">
-<p rend="indent">ἄλλη ἀκούσασα περὶ τοῦ υἱοῦ, ὡς κακῶς ἐπὶ τῆς ξένης ἀναστρέφοιτο, ἔγραψε, <q type="unspecified">κακά τευ<note xml:lang="eng" anchored="true" place="unspecified" resp="ed">
-<foreign xml:lang="grc">τευ</foreign> Valckenaer as in no. 3 <foreign xml:lang="lat">supra</foreign> (241 a) and Stobaeus, <title xml:lang="lat">Florilegium</title>, cviii. 83: <foreign xml:lang="grc">τοι</foreign>.</note> φάμα κακκέχυται· ταύταν ἀπώθευ<note xml:lang="eng" anchored="true" place="unspecified" resp="ed">
-<foreign xml:lang="grc">ἀπώθευ</foreign> Hatzidakis (<foreign xml:lang="lat">cf.</foreign> Cicero, <title xml:lang="lat">Ad Atticum</title>, xiv. 22): <foreign xml:lang="grc">ἀπόθευ</foreign>.</note> ἢ μὴ ἔσο.</q>
+<div type="textpart" subtype="section" n="10">
+<p>ἄλλη ἀκούσασα περὶ τοῦ υἱοῦ, ὡς κακῶς ἐπὶ τῆς ξένης ἀναστρέφοιτο, ἔγραψε, <q type="unspecified">κακά τευ<note resp="ed" place="unspecified" anchored="true">
+<foreign xml:lang="grc">τευ</foreign> Valckenaer as in no. 3 <foreign xml:lang="lat">supra</foreign> (241 a) and Stobaeus, <title xml:lang="lat" rend="italic">Florilegium</title>, cviii. 83: <foreign xml:lang="grc">τοι</foreign>.</note> φάμα κακκέχυται· ταύταν ἀπώθευ<note resp="ed" place="unspecified" anchored="true">
+<foreign xml:lang="grc">ἀπώθευ</foreign> Hatzdakis (<foreign xml:lang="lat">cf.</foreign> Cicero, <title xml:lang="lat" rend="italic">Ad Atticum</title>, xiv. 22): <foreign xml:lang="grc">ἀπόθευ</foreign>.</note> ἢ μὴ ἔσο.</q>
```

Comment:
looks like the name spelling was inadvertently changed from Hatzidakis to Hatzdakis